### PR TITLE
RT/YJ/Update system type RMDs

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
While I worked on the ```is_baseline_system``` functions and their pytest development, I realized the system type RMDs weren't up-to-date. Therefore, I put the latest system type RMDs here. 

Newly added RMDs are
1. system 3-(a), (b), (c) - These weren't modeled so, I modeled and included them in this PR.
2. system 8-(a), (b), (c) weren't included in the ```develop``` branch. So, I also added them here.

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/system_types/System 12_CAV_SZ_HW.json [hunk 1]
```diff
@@ -95,13 +90,7 @@
...
                     "id": "Chiller Loop 1",
-                    "type": "COOLING",
-                    "child_loops": [
-                        {
-                            "id": "Secondary CHW Loop 1",
-                            "type": "COOLING"
-                        }
-                    ]
+                    "type": "COOLING"
                 }
```
Comment (jugonzal07) [inline rct229/ruletest_engine/ruletest_jsons/system_types/System 12_CAV_SZ_HW.json:94]: I'm unsure why the child loops are being removed. I wonder if the system_types excel file on onedrive is out of date? It appears different than what I have locally and could be the source of this. You'll need the new script I uploaded in this PR to generate the JSONs from the spreadsheet, I believe:
https://github.com/pnnl/ruleset-checking-tool/pull/835

Comment (yunjoonjung-PNNL) [reply rct229/ruletest_engine/ruletest_jsons/system_types/System 12_CAV_SZ_HW.json]: Juan, as we discussed with each other, the system type master Excel file was updated based on your latest version (which includes the child loops). Could you check this out one more time?

Also, the system type master Excel file on OneDrive was updated.

**Review by weilixu — APPROVED** [review]
Great work, thanks, Yun Joon. One question, does the excelsheets updated as well?

**Comment by yunjoonjung-PNNL** [comment]
> Great work, thanks, Yun Joon. One question, does the excelsheets updated as well?

Thank you Weili! Yes, the Excel file on OneDrive is updated as well.

**Review by weilixu — APPROVED** [review]

**Review by jugonzal07 — CHANGES_REQUESTED** [review]
I noticed these two terminals used different reheat systems. Could you look into this?

https://github.com/pnnl/ruleset-checking-tool/blob/6e7d9c9a27461302a84cbd012a1581f5a2d3c68a/rct229/ruletest_engine/ruletest_jsons/system_types/System_6b_PVAV_Elec_Reheat.json#L20-L39

**Comment by yunjoonjung-PNNL** [comment]
> I noticed these two terminals used different reheat systems. Could you look into this?
> 
> https://github.com/pnnl/ruleset-checking-tool/blob/6e7d9c9a27461302a84cbd012a1581f5a2d3c68a/rct229/ruletest_engine/ruletest_jsons/system_types/System_6b_PVAV_Elec_Reheat.json#L20-L39

Great catch! This was fixed and fan_config was also added because this was missing in the past.

**Review by jugonzal07 — APPROVED** [review]

**Pull request merged**